lti_consumer/templates/html/lti_iframe.html

```diff
@@ -1,8 +1,9 @@
+<%page args="launch_url" expression_filter="h"/>
```

Comment:
@douglashall Thanks for the explanation.  It might have been just that I was tired, but my brain wasn't able to easily understand why this was needed when we had data-launch-url.

I'm not sure how much clarity this would add, but would you mind renaming this page arg to "default_launch_url" or "initial_launch_url"?  Does that make sense to you?